src/archiefvernietigingscomponent/sass/components/_review-create.scss

```diff
@@ -20,4 +20,24 @@
   @include element("list-details") {
     color: $color-text--light;
   }
+
+  @include element("review-comment") {
+    padding: 1em 1em;
```

Comment:
Same remark as the above:

* left and right padding should be the same as in other elements (feedback field and the table)
* Author comment and review feedback should be separated from each other since they are different kinds of data (non-interactive and interactive) 
![image](https://user-images.githubusercontent.com/47745906/109509742-82571300-7aa1-11eb-9828-300e5f8f02a1.png)

Reply:
That is strange :thinking: This is how it looks for me?
![Screenshot from 2021-03-02 09-32-51](https://user-images.githubusercontent.com/19154114/109620610-5d62ae80-7b3a-11eb-98e4-3fc9f2c4f57b.png)

Reply:
Hmm I ran `gulp build` one more time and now it looks the same for me :)